scaleout/stackn/templates/event-listener-deployment.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       serviceAccountName: studio
       containers:
       - image: {{ .Values.eventListener.image.repository }}
```

Comment:
for the main container add:
{{- if .Values.studio.securityContext.enabled }}
        securityContext:
          runAsUser: {{ .Values.studio.securityContext.runAsUser }}
          runAsGroup: {{ .Values.studio.securityContext.runAsGroup }}
          allowPrivilegeEscalation: {{ .Values.studio.securityContext.allowPrivilegeEscalation }}
          privileged: {{ .Values.studio.securityContext.privileged }}
          capabilities:
            drop:
              - all
        {{- end }}
        {{- if .Values.studio.readinessProbe.enabled }}

Reply:
Alright man, will do!